src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -10,6 +10,9 @@ state_update_interval_sec = 5
 min_controller_id = {{ spec.min_controller_id }}
 max_controller_id = {{ spec.max_controller_id }}
 enable_spdk_discovery_controller = {{ spec.enable_spdk_discovery_controller }}
+enable_prometheus_exporter = True
+prometheus_exporter_ssl = False
+prometheus_port = 10008
```

Comment:
while currently we don't allow this to be changed, for future proofing reasons, we might want to have this be a variable passed to the context for generating this file rather than just a hardcoded value. Something like (in `prepare_create` for the NvmeofService class)
```
context = {
            'spec': spec,
            'name': name,
            'addr': host_ip,
            'port': spec.port,
            'log_level': 'WARN',
            'rpc_socket': '/var/tmp/spdk.sock',
            'transport_tcp_options': transport_tcp_options,
            'rados_id': rados_id,
            'prometheus_port' : self.PROMETHEUS_PORT      <--- THIS LINE
        }
```
and then we can populate the conf with that value here.

Reply:
Sounds good to me, probably we can create a tracker for configuring this & also I think adding ssl support?